usaspending_api/accounts/models.py

```diff
@@ -19,6 +19,8 @@ class FederalAccount(models.Model):
     agency_identifier = models.TextField(db_index=True)
     main_account_code = models.TextField(db_index=True)
     account_title = account_title = models.TextField()
+    # agency_identifier + '-' + main_account_code + ' - ' + account_title
```

Comment:
Some additional text here like "The federal_account_code field is made up of: ..." will make it clear it's describing the field below it